templates/partials/sidebar.html

```diff
@@ -0,0 +1,19 @@
+<aside class="sidebar flex-sidebar">
+
+        <nav class="book-menu uk-margin-bottom">
+
+            {% url 'book_list' as book_list %}
+            {% url 'book_new' as book_new %}
+            <ul class="uk-nav nav-books flex-nav-books flex-to-block-l">
+
+            <li class="uk-button uk-button-primary pad-15
+             {% if request.get_full_path == book_list %}  active {% endif %}"><a href="{% url 'book_list' %}">All books</a></li>
+
+            <li class="uk-button uk-button-primary pad-15
+            {% if request.get_full_path == book_new %} active {% endif %}"
+            ><a href="{% url 'book_new' %}">Add a book</a></li>
```

Comment:
Can you use just `{{ book_new}}` here since you're predeclaring it above? Or remove the `{% url ... %}` on lines 5-6 if unused!

Reply:
5 and 6 are needed to check because in e.g. line 12 it would not accept the url ... syntax. But you are right, I can use {{ book_new }} as href, and also {{ book_list  }}! Will go in next PR, too :)